packages/rocketchat-livechat/client/lib/Analytics.js

```diff
@@ -0,0 +1,116 @@
+RocketChat.Livechat = {};
+
+class Analytics {
```

Comment:
What about to use the import/export approach to `Analytics` lib?
IMO, you can just create an object, like `export const Analytics = {..` and implement the analytics methods(props) inside it, what do you think?
Using this approach you just need to `import` this object in `Rocket.Chat/packages/rocketchat-livechat/server/lib/Livechat.js`, like bellow:

```/* globals HTTP */
import _ from 'underscore';
import s from 'underscore.string';
import UAParser from 'ua-parser-js';
import LivechatVisitors from '../models/LivechatVisitors';
import { Analytics } from Analytics;

RocketChat.Livechat = {
        Analytics,
	historyMonitorType: 'url',

	logger: new Logger('Livechat', {
		sections: {
			webhook: 'Webhook'
		}
	}),```

Reply:
We need this lib on client, while `Rocket.Chat/packages/rocketchat-livechat/server/lib/Livechat.js` is server side lib only. So, I guess this is the it would work.